src/batch/parquet-schema.ts

```diff
@@ -14,18 +28,26 @@ export const BroadcastSchema = {
   signature: { type: "BYTE_ARRAY" },
 };
 
+export const BroadcastBloomFilterOptions: BloomFilterOptions = {
```

Comment:
Note here in all these bloom filter options, you can opt to add any extras; this const and the others simple ensures we have the ones we want, i.e. standardizes on the columns to add to the bloom filter,  based on the spec.